packages/landing-app/src/index.scss

```diff
@@ -24,28 +26,22 @@ body {
 
 .pt-container {
   margin: 0 auto;
-  max-width: $pt-grid-size * 96;
+  max-width: $container-width;
   padding: 0 $pt-grid-size * 2;
 }
 
 header {
   position: relative;
   margin: 0;
   background-color: transparent;
```

Comment:
@llorca maybe set an actual color here to prevent that flash of white before first canvas render?

Reply:
the FOUC isn't just in the header, it's the whole page. not sure how to address that